tests/datatypes/test_str.py

```diff
@@ -517,6 +517,33 @@ def test_rindex(self):
                 print(err)
             """)
 
+    def test_rjust(self):
+        # test cases to generate valid outout
+        self.assertCodeExecution("""
+            st="a good cook could cook good"
+            print(st.rjust(100))
+            print(st.rjust(100, 'X'))
+            print(st.rjust(10, 'X'))
+
+            """)
+
+        # test cases to generate exception
+        self.assertCodeExecution("""
+            st="a good cook could cook good"
+            try:
+                print(st.rjust(-20, 'cook'))
```

Comment:
Does this trigger the edge case for negative width or for too-long fill char? It might be better to have those as separate tests, to make sure both cases get exercised.